xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -585,7 +585,12 @@ div.userInfo, div.userPreferences, div.watchlistManagement, div.userDashboard {
 .userInfo img {
   max-width: 100%;
 }
-
+        
+.userInfo h2 {
+  font-size: 16px;
```

Comment:
Using absolute values is not a good practice in general, and especially in the case of font size. Moreover, pixels is not a great unit to specify the font size. Try a relative value / unit. How does it look for instance with:

```
font-size: larger;
font-weight: bolder;
```

Reply:
I replace the `font-size` value from `px` to `large`. On relative value `large`. It gives the same result as above in the screenshot.

Reply:
``large`` is not a relative value. ``larger`` is. See https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#Syntax .